scss/_spinners.scss

```diff
@@ -0,0 +1,83 @@
+//
+// Rotating border
+//
+
+@keyframes spinner-border {
+  to { transform: rotate(360deg); }
+}
+
+.spinner-border {
+  position: relative;
+  display: inline-block;
+  width: 2rem;
+  height: 2rem;
+  overflow: hidden;
+  text-indent: -999em;
+  vertical-align: text-bottom;
+  border: .25em solid;
+  border-color: currentColor transparent currentColor currentColor;
+  border-radius: 50%;
+  animation: spinner-border .75s linear infinite;
+}
+
+.spinner-border-reverse {
+  border-color: transparent currentColor transparent transparent;
+
+  &::after {
```

Comment:
I can't see any difference in Chromium 69 and Firefox Nightly 64 when not using `::after` section.

For me `.spinner-border-reverse` should just invert spinners `border-color` property, or am i wrong?. Perhaps this a browser specific thing, like for IE10+?